tests/testthat/test-DataWrangling.R

```diff
@@ -676,7 +725,11 @@ test_that("WrangleUploadData() wrangles continuous wide data to be usable in the
   data$T.1 <- c(1, 2, 1)
   data$T.2 <- c(3, 1, 4)
   data$T.3 <- c(5, 6, NA)
-  data <- data[order(data$StudyID, data$T.1, data$T.2, data$T.3), ]
+  #Local function for sorting long data
+  SortLong <- function(long_data) {
+    long_data[order(long_data$StudyID, long_data$T), ]
+  }
```

Comment:
In `DataWrangling.R`, if you define `SortLong` as its own function (i.e. outside of `SortByStudyIDThenT`), then you should be able to use it in the tests without having to redefine it each time